js_modules/dagster-ui/packages/ui-core/src/partitions/__tests__/OrdinalOrSingleRangePartitionSelector.test.tsx

```diff
@@ -110,69 +107,4 @@ describe('OrdinalOrSingleRangePartitionSelector', () => {
       '{"selectedKeys":["CA","MN"],"selectedRanges":[]}',
     );
   });
-
-  it('should support All, Single or Range entry for time partition dimensions', async () => {
```

Comment:
could just do `it.skip` and add a comment to ignore.

Reply:
looks like we lint error on disabled test 
if we want it back its always here in git history